.github/actions/image-builder/README.md

```diff
@@ -0,0 +1,117 @@
+# Image Builder Action
+
+This action is supposed to trigger image-builder service. It receives authentication tokens (**oidc** and **azure personal access token**) via inputs as well as image name.
+
+Optionally you can set other inputs to control which image with which context should be built.
+
+# Inputs
+
+## **oidc-token**
+
+JWT token exposed by **expose-jwt** github action, which is JWT token signed by github, containing information about running workflow.
+
+Required: **true**
+
+## **ado-token**
+
+Azure DevOps personal access token (PAT), which is used by this action to authenticate against Azure API.
+
+Required: **true**
+
+## **image-name**
+
+Name of the image that will be build via image builder.
+
+Required: **true**
+
+## **context**
+
+Optional parameter that allows you to set build context for image builder, related to the root of the repository.
+
+Default: **'.'**
+
+## **dockerfile**
+
+Optional parameter that allows you to set path to the Dockerfile for the image to build.
+
+Default: **'Dockerfile'**
+
+## **build-args**
+
+Optional parameter that allows you to set additional build arguments.
+Each argument has format `name=value`, multiple arguments are separated by new line.
+
+Default: **''**
+
+## **tags**
+
+Optional parameter that allows you to pass tags with which image will be build. Each tag is placed in the new line.
+
+Default: **''**
+
+## **export-tags**
+
+Optional parameter that enable exporting tags provided via **tags** input and default tags to be exported as build argument.
+Each tag will have format **TAG_x**, where `x` is the tag name passed along with the tag.
+
+Default: **false**
+
+## **config**
+
+Optional parameter which sets the config file containing information about connection to the Azure DevOps.
+
+Default: **'./configs/kaniko-build-config.yaml'**
+
+## **env-file**
+
+Optional parameter that allows you to provide path to the file with environment variables which loads in the build
+
+Default: **''**
+
+## **dry-run**
+
+Optional parameter that allows you to prevent calling Azure service
```

Comment:
We need an article, but I don't understand the context enough to know which
Either
```suggestion
Optional parameter to prevent calling an Azure service
```
Or
```suggestion
Optional parameter to prevent calling the Azure service
```